nf_core/pipeline-template/.github/workflows/awstest.yml

```diff
@@ -24,5 +24,5 @@ jobs:
             {
               "outdir": "s3://{% raw %}${{ secrets.AWS_S3_BUCKET }}{% endraw %}/{{ short_name }}/{% raw %}results-${{ github.sha }}{% endraw %}"
```

Comment:
```suggestion
              "outdir": "s3://{% raw %}${{ secrets.AWS_S3_BUCKET }}{% endraw %}/{{ short_name }}/{% raw %}results-small-tests-${{ github.sha }}{% endraw %}"
```

Reply:
Just thought about this now, that avoids conflicts with big tests commit sha!

Reply:
Ah, crap! So they would be put in the same directory if we don't change that?